internal/render/pod.go

```diff
@@ -73,6 +73,15 @@ func (p Pod) ColorerFunc() model1.ColorerFunc {
 		case Running:
 			if c != model1.ErrColor {
 				c = model1.StdColor
+
+				idx, ok = h.IndexOf("READINESS GATES", true)
+				if ok {
+					rg := strings.TrimSpace(re.Row.Fields[idx])
```

Comment:
I think it would be best to add this check in the diagnose section. This could confuse users as the pod will indicate issues and yet no explanation.  What do you think?